### PR TITLE
The student course registration has been implemented successfully

### PR DESCRIPTION
**Pull request by z-alex**
Task 5,1 ISSUES #152  #121 

# **Main changes:**
- A drop down menu for student course registration selection (all courses in the DB are shown)
- The description of the selected course is displayed
- Add the course in the database when the submit button is clicked
- Display all courses of the student in the student dashboard

### **Edit:**
- Added a fix to prevent a student from registering to the same course twice

# **Other changes:**
- The course buttons redirect the user to the course dashboard. ISSUES #139 #124

### CODE REVIEW

**Comment by Helen2hang** [comment]
Pair programming with Alex,

File: pages/courseRegistration.php [hunk 1]
```diff
@@ -4,9 +4,14 @@
...
         <div class="container-fluid dashboardDiv  text-center">
-        <!-- Insert what you want inside dashboard here-->
-        <h1>Course Registration</h1>
-        Under construction
+            <!-- Insert what you want inside dashboard here-->
+            <h1>Course Registration</h1>
+            Select a course to register:
+            <select id="course_selection">
+                <!-- <option value="volvo">Volvo</option> -->
+            </select>
```
Comment (Helen2hang) [inline pages/courseRegistration.php]: The CSS for this part should be done by @weieiei

**Comment by Helen2hang** [comment]
@z-alex You should add a fix that won't allow the student to register to the same course more than once.

**Comment by z-alex** [comment]
@Helen2hang done

**Comment by z-alex** [comment]
Now the course buttons redirect the user to the corresponding course dashboard.
before
capture

File: pages/courseRegistration.php (hunk 1 above)
Comment (Weieiei) [inline pages/courseRegistration.php]: Ok I will do it soon

**Comment by orelmoisa** [comment]
@z-alex I think the student should be able to see the course description before they register

**Comment by Helen2hang** [comment]
I agree with Orel

**Comment by z-alex** [comment]
@orelmoisa Okay, I'll add this feature now.

**Comment by z-alex** [comment]
@orelmoisa Done. The description changes dynamically depending on the selected option from the drop down menu.
capture

**Comment by orelmoisa** [comment]
@z-alex perfect!!

**Pull request merged**